app/views/verification/letter/new.html.erb

```diff
@@ -50,7 +50,7 @@
       <ul>
         <li>
           <%= t("verification.letter.new.office",
-          office: link_to(t("verification.letter.new.offices"), t("verification.letter.new.offices_url"),
+          office: link_to(t("verification.letter.new.offices"), Setting['verification_offices_url'],
```

Comment:
Please use `setting['verification_offices_url']` instead. 
There is a `setting` helper method to use in views and have the settings memoized per request.